src/log/list.js

```diff
@@ -15,22 +14,15 @@ const { removeUndefinedKeys } = require('../utils/objUtils')
  * @param {string} params.processId
  * @param {number} params.limit
  * @param {number} params.startTime
- * @param {number} params.endTime
+ * @param {string} params.level
+ * @param {string} params.subject
+ * @param {number} params.skip
```

Comment:
might as well take the opportunity to clean up the docs here because some keys are beeing deprecated with the next prs and new ones are beeing added, detailed docs shouldnt be part of the sdk (since they change more often this would need a version update)
`
 * @param {Object} params
 * @param {Object} params.queryStringParams
`

this way the docs will not be wrong and further information can always be found in the actual endpoint docs